### PR TITLE
Add several spelling corrections.

### PR DESCRIPTION
**Pull request by flopp**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3732,6 +3733,8 @@ defauts->defaults
 defecit->deficit
 defenate->definite
 defenately->definitely
+defence->defense
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This is an American/British thing.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -7837,6 +7841,7 @@ libreoffie->libreoffice
 libreoficekit->libreofficekit
 libstc++->libstdc++
 licenceing->licencing
+licences->licenses
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Again

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: See also https://github.com/codespell-project/codespell/pull/286

**Comment by flopp** [comment]
Ok, I see. I'll remove the US/GB differences.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5151,6 +5152,7 @@ equivilent->equivalent
 equivlalent->equivalent
 equvalent->equivalent
 erally->orally, really,
+erasuer->erasure
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: A bit late, but I'd also add eraser as a suggestion.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: https://github.com/codespell-project/codespell/commit/47a8ebdc8903de81c501ba3e229daaf258544dca

**Pull request merged**